### PR TITLE
Reorganize mailing lists : steering and code of conduct committee

### PR DESCRIPTION
**Pull request by ameukam**
Move mailing lists for the Steering Committee and the Code of Conduct
Committee to dedicated folders.

Signed-off-by: Arnaud Meukam <ameukam@gmail.com>

### CODE REVIEW

File: groups/committee-code-of-conduct/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+    committee-code-of-conduct
```
Comment (spiffxp) [inline groups/committee-code-of-conduct/OWNERS:4]: No approvers until we can ensure subdirs of groups can't create arbitrary groups

File: groups/steering/OWNERS [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+options:
+    no_parent_owners: true
+
+approvers:
+    committee-steering
```
Comment (spiffxp) [inline groups/steering/OWNERS:7]: No approvers until we can ensure subdirs of groups can't create arbitrary groups
```suggestion
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
Really need to get some kind of `restrictions.yaml` setup or some other way to ensure that group creation requires root approval.

File: groups/committee-code-of-conduct/OWNERS (hunk 1 above)
Comment (ameukam) [reply groups/committee-code-of-conduct/OWNERS]: Fixed.

File: groups/steering/OWNERS (hunk 2 above)
Comment (ameukam) [reply groups/steering/OWNERS]: Fixed.

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**